components/03-modules/side-menu/SideMenu.html

```diff
@@ -0,0 +1,43 @@
+<ul class="side-menu">
+  <li
+    v-if="prevLevel"
+    class="side-menu__item side-menu__item--prev"
+  >
+    <button
+      type="button"
+      class="side-menu__button side-menu__button--prev"
+      @click.prevent="load(prevLevel)"
+    >
+      <alpaca-icon
+        icon="angle-left"
+        class="side-menu__icon side-menu__icon--prev"
+      />
+      {{ currentItem.label }}
```

Comment:
indentation

Reply:
Seems to be ok right now, it's on the same level with self-closed `alpaca-icon` component